src/components/MoneyInput/index.js

```diff
@@ -34,6 +34,10 @@ const SelectWrapper = styled.div`
   }
 `
 
+const StyledSelect = styled(Select)`
+  cursor: default;
+`
```

Comment:
I think it’s better to add this logic in the `Select` component instead. Also I assume you only want the default cursor when it’s disabled?

```scss
&:[disabled] {
  cursor: default;
}
```

Reply:
Ah yes pushed too soon 😅but that is a better solution yes.